exim/init.sls

```diff
@@ -50,4 +52,4 @@ exim_install_packages:
 
 exim_service:
   service.running:
-    - name: exim4
+    - name: {{ map.service }}
```

Comment:
Could you open a separate PR for this, please? That way we don't have to mix the two issues addressed (split config, service name). Thank you!